### PR TITLE
fix: Cannot import existing configuration 

### PR DESCRIPTION
**Pull request by raosan**
implement issue #53 by:
- fix: error, probe name & desc, and default check alerts
- show error if file not valid (default file is json file. user cannot upload other than json file, so the validation is just checking is the file have probes and notifications key)

### Notes:

For more advanced validation in the future, we can use `validateConfig` function from monika, but for now it cannot be used.

### CODE REVIEW

**Comment by nicnocquee** [comment]
# Bug

When there is body in a request in the imported configuration, the Body option stays No Body.

Screenshot 2021-04-19 at 18 17 13

**Comment by raosan** [comment]
> # Bug
> When there is body in a request in the imported configuration, the Body option stays No Body.
> 
> Screenshot 2021-04-19 at 18 17 13

fixed. please check

File: contexts/importer.ts [hunk 1]
```diff
@@ -22,8 +22,17 @@ export const useConfigFileImporter = () => {
...
+          if (probes) {
+            handleSetProbes(probes);
+          } else {
+            throw new Error('something wrong in probes key');
+          }
+
+          if (notifications) {
+            handleSetNotifications(notifications);
+          } else {
+            throw new Error('something wrong in notifications key');
```
Comment (armedi) [inline contexts/importer.ts:34]: The notifications field is not a mandatory for monika, so I think this line should not throw error

reference: https://github.com/hyperjumptech/monika/blob/main/src/interfaces/config.ts

**Review by armedi — COMMENTED** [review]
When I try to import the basic config provided in https://github.com/hyperjumptech/monika/blob/main/monika.example.json it throws error.

This is because many of the fields are assumed to exist in the probe interface which is imported from [@hyperjumptech/monika/lib/interfaces/probe](https://github.com/hyperjumptech/monika/blob/main/src/interfaces/probe.ts), while most of them are actually allowed to be empty. Can we also use [sanitizeProbe function from monika](https://github.com/hyperjumptech/monika/blob/main/src/components/looper.ts#L33) here?

File: contexts/importer.ts (hunk 1 above)
Comment (raosan) [reply contexts/importer.ts]: fixed

**Comment by raosan** [comment]
> When I try to import the basic config provided in https://github.com/hyperjumptech/monika/blob/main/monika.example.json it throws error.
> 
> This is because many of the fields are assumed to exist in the probe interface which is imported from [@hyperjumptech/monika/lib/interfaces/probe](https://github.com/hyperjumptech/monika/blob/main/src/interfaces/probe.ts), while most of them are actually allowed to be empty.

fixed. please check

**Review by nicnocquee — APPROVED** [review]

**Review by armedi — APPROVED** [review]

**Pull request merged**